ibu-imager/cmd/create.go

```diff
@@ -88,6 +89,14 @@ func create() error {
 	op := ops.NewOps(log, hostCommandsExecutor)
 	rpmOstreeClient := ostree.NewClient("ibu-imager", hostCommandsExecutor)
 
+	defer func() {
+		if err = seedrestoration.NewSeedRestoration(log, op, common.BackupDir, containerRegistry,
```

Comment:
I wanted to provide users with the possibility of just re-run the imager directly if they hit an error (e.g., containers failing to stop). In this case, imager would handle cleanup for them and let the seed cluster ready for creating a seed image again. But I keep open to suggestions / changes here.

Reply:
We need the auto cleanup for end-to-end seed generation orchestration from LCA, even on failure, otherwise LCA won't come back up. So it would at least need to be an option that can be passed on the cmdline to always do cleanup/recovery